packages/ove-asset-manager/src/OVE.Service.AssetManager/Domain/IAssetFileOperations.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using OVE.Service.AssetManager.Models;
+
+namespace OVE.Service.AssetManager.Domain {
+    public interface IAssetFileOperations {

```

Comment:
Are these operations on Asset Files or Asset Models?

Reply:
These methods operate on the files of assets. 
please suggest a new name if preferred.